notebooks/01-mnist-hello-world.ipynb

```diff
@@ -0,0 +1,401 @@
+        "\n",
+        "---\n",
+        "  - Give us a ⭐ [on Github](https://www.github.com/PytorchLightning/pytorch-lightning/)\n",
+        "  - Check out [the documentation](https://pytorch-lightning.readthedocs.io/en/latest/)\n",
+        "  - Join us [on Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-f6bl2l0l-JYMK3tbAgAmGRrlNr00f1A)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2LODD6w9ixlT",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Setup  \n",
+        "Lightning is easy to install. Simply ```pip install pytorch-lightning```"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "zK7-Gg69kMnG",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "! pip install pytorch-lightning --quiet"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "w4_TYnt_keJi",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import os\n",
+        "\n",
+        "import torch\n",
+        "from torch import nn\n",
+        "from torch.nn import functional as F\n",
+        "from torch.utils.data import DataLoader, random_split\n",
+        "from torchvision.datasets import MNIST\n",
+        "from torchvision import transforms\n",
+        "import pytorch_lightning as pl\n",
+        "from pytorch_lightning.metrics.functional import accuracy"
+      ],
+      "execution_count": 2,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EHpyMPKFkVbZ",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Simplest example\n",
+        "\n",
+        "Here's the simplest most minimal example with just a training loop (no validation, no testing).\n",
+        "\n",
+        "**Keep in Mind** - A `LightningModule` *is* a PyTorch `nn.Module` - it just has a few more helpful features."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "V7ELesz1kVQo",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "class MNISTModel(pl.LightningModule):\n",
+        "\n",
+        "    def __init__(self):\n",
+        "        super(MNISTModel, self).__init__()\n",
+        "        self.l1 = torch.nn.Linear(28 * 28, 10)\n",
+        "\n",
+        "    def forward(self, x):\n",
+        "        return torch.relu(self.l1(x.view(x.size(0), -1)))\n",
+        "\n",
+        "    def training_step(self, batch, batch_nb):\n",
+        "        x, y = batch\n",
+        "        loss = F.cross_entropy(self(x), y)\n",
+        "        return pl.TrainResult(loss)\n",
```

Comment:
maybe we can simply return the loss here, and explain the trainResult object in the second example?

Reply:
good eye..might be good idea. I'll do a little more cleanup after initial merge. don't want to mess with json files too much right now so I avoid breaking things.